.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.token
```

Comment:
I'd prefer to use env rather than `.token` files, for the reasons discussed by @wchargin here: [1]

[1]: https://github.com/sourcecred/sourcecred/pull/1288#discussion_r314375371

Reply:
Missed that as it was resolved already there. Removed those.

Reply:
A comment from https://github.com/sourcecred/sourcecred/pull/1288 that I'll include here too - instead of doing an export it might be advisable to just one off define the variable for each command:

```bash
SOURCECRED_GITHUB_TOKEN=YOUR_GITHUB_TOKEN docker run --rm -ti -v sourcecred_data:/data -e SOURCECRED_GITHUB_TOKEN sourcecred load sourcecred/sourcecred
```
I'm okay with the export (I don't share a system with anyone), but this was suggested for security.

Still testing :)

Reply:
okay one quick point of feedback - there isn't any instruction to build the container, so I get to the line where I need `sourcecred-widgets` and I'm like :thinking: huh? We should add this, before it is needed:

Next, build the "sourcecred-widgets" container from the Dockerfile in the repository:

```bash
$ docker build -t sourcecred-widgets .
```

Reply:
Also we still need to have the GitHub token, so that should be added to the example command.

Reply:
It's so beautiful!! 

![image](https://user-images.githubusercontent.com/814322/63170365-1ddf7a80-c007-11e9-9585-b7c96d2ae326.png)

Screenshot of an svg, sorry about that :)

@Beanow when we get this merged, I can do another automated build for it, and then guess what! We can make GitHub actions too! I'm so pumped.